From b4f5f6b753c4ea6917c6488c9d90ccc473e4067c Mon Sep 17 00:00:00 2001
From: Iain Sandoe <iain@sandoe.co.uk>
Date: Thu, 30 Jul 2020 19:00:55 +0100
Subject: [PATCH] Darwin, Arm64 : Top level boiler-plate changes.

This just adds the target to the top level configure. NOTE that
we allow arm64 and aarch64 here.

(cherry picked from commit dac80a7c4b1f6919fc7dbd876b336da2ec2957d9)
---
 configure    | 4 +++-
 configure.ac | 4 +++-
 2 files changed, 6 insertions(+), 2 deletions(-)

diff --git configure configure
index a9b41249686..9367a6db04b 100755
--- configure
+++ configure
@@ -3740,7 +3740,7 @@ esac
 case "${target}" in
   *-*-chorusos)
     ;;
-  aarch64-*-darwin*)
+  aarch64-*-darwin* | arm64*-*-darwin*)
     noconfigdirs="$noconfigdirs ld gas gdb gprof"
     noconfigdirs="$noconfigdirs sim target-rda"
     ;;
@@ -4085,6 +4085,8 @@ fi
   hppa*-*)
     host_makefile_frag="config/mh-pa"
     ;;
+  aarch64-*-darwin* | arm64*-*-darwin*)
+    ;;
   *-*-darwin*)
     host_makefile_frag="config/mh-darwin"
     ;;
diff --git configure.ac configure.ac
index 38b61964f0f..079259b5365 100644
--- configure.ac
+++ configure.ac
@@ -1007,7 +1007,7 @@ esac
 case "${target}" in
   *-*-chorusos)
     ;;
-  aarch64-*-darwin*)
+  aarch64-*-darwin* | arm64*-*-darwin*)
     noconfigdirs="$noconfigdirs ld gas gdb gprof"
     noconfigdirs="$noconfigdirs sim target-rda"
     ;;
@@ -1333,6 +1333,8 @@ case "${host}" in
   hppa*-*)	
     host_makefile_frag="config/mh-pa"
     ;;
+  aarch64-*-darwin* | arm64*-*-darwin*)
+    ;;
   *-*-darwin*)
     host_makefile_frag="config/mh-darwin"
     ;;
-- 
2.40.1

